src/locales/en/translation.json

```diff
@@ -314,6 +314,22 @@
         }
       }
     },
+    "pyiTriageSelectDevice": {
+      "title": "Are you on a computer or tablet right now?",
+      "header": "Are you on a computer or tablet right now?",
+      "content": {
+        "paragraph1": "Some ways of confirming your identity require a smartphone. Let us know what you're using right now.",
+        "formRadioButtons": {
+          "onComputerOrTabletButtonText": "Yes, I am on a computer or tablet",
+          "onSmartphoneButtonText": "No, I am on a smartphone"
+        },
+        "formErrorMessage": {
+          "errorSummaryTitleText": "There is a problem",
+          "errorSummaryDescriptionText": "Select what you would like to do",
```

Comment:
Has this error text come from UCD? I would expect it to be something like `Select whether you are on a smartphone or computer`. We shouldn't just use the same error message for everything.

Reply:
Validation is out of scope of this ticket really and we haven't had any error content from UCD yet (though probably fine to leave this as is for now - there's a follow up ticket [PYIC-5254](https://govukverify.atlassian.net/browse/PYIC-5254) to update finalised content)

Reply:
In general, the text comes from DCMAW, since we haven't received confirmation from UCD. I picked this up from another page template but I see your suggestion is a better fit 👍